### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by wagenet**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,22 +21,25 @@
...
-    "tslint": "^5.11.0"
+    "lolex": "^6.0.0",
+    "qunit": "^2.14.1",
+    "rollup-plugin-buble": "^0.19.8",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "tslint": "^6.1.3"
+  },
+  "engines": {
+    "node": "10.* || >= 12.*"
```
Comment (rwjblue) [inline package.json:43]: Why is this needed?

Comment (wagenet) [reply package.json]: Removed.
